client/src/App.js

```diff
@@ -1,11 +1,64 @@
+import React, { useState } from "react";
 import "./App.css";
+import data from "./exampleresponse.json";
+import VideoList from "./VideoList";
+import AddVideo from "./AddVideo";
+import SearchVideo from "./SearchVideo";
 
 function App() {
+  const [VideosData, setVideosData] = useState(data);
+  const [searchVideos, setSearchVideos] = useState([]);
+  console.log(VideosData);
+
+  function addVideo(title, url) {
+    let newVideo = {
+      id: Math.floor(Math.random() * 1000000),
```

Comment:
Better to use a dedicated ID generating package (e.g. `uuid`), but if you're replacing this with a full-stack version (e.g. where you let the database handle the ID) then it's fine as a stop-gap.